templates/admin/payment-actions.php

```diff
@@ -1,40 +1,43 @@
 <?php
+/** @var WC_Gateway_Swedbank_Pay_Cc $gateway */
 /** @var WC_Order $order */
 /** @var int $order_id */
 /** @var string $payment_id */
 /** @var array $info */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-} // Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
 
 ?>
 <div>
-	<strong><?php _e( 'Payment Info', WC_Swedbank_Pay::TEXT_DOMAIN ) ?></strong>
-	<br />
-	<strong><?php _e( 'Number', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>:</strong> <?php echo esc_html( $info['payment']['number'] ); ?>
-	<br />
-	<strong><?php _e( 'Instrument', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>: </strong> <?php echo esc_html( $info['payment']['instrument'] ); ?>
-	<br />
-	<strong><?php _e( 'Intent', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>: </strong> <?php echo esc_html( $info['payment']['intent'] ); ?>
-	<br />
-	<strong><?php _e( 'State', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>: </strong> <?php echo esc_html( $info['payment']['state'] ); ?>
-	<br />
-	<?php if ( isset($info['payment']['remainingCaptureAmount']) && (float) $info['payment']['remainingCaptureAmount'] > 0.1 ): ?>
+	<strong><?php _e( 'Payment Info', 'swedbank-pay-woocommerce-payments' ); ?></strong>
+	<br/>
+	<strong><?php _e( 'Number', 'swedbank-pay-woocommerce-payments' ); ?>
+		:</strong> <?php echo esc_html( $info['payment']['number'] ); ?>
+	<br/>
+	<strong><?php _e( 'Instrument', 'swedbank-pay-woocommerce-payments' ); ?>
+		: </strong> <?php echo esc_html( $info['payment']['instrument'] ); ?>
+	<br/>
+	<strong><?php _e( 'Intent', 'swedbank-pay-woocommerce-payments' ); ?>
+		: </strong> <?php echo esc_html( $info['payment']['intent'] ); ?>
+	<br/>
+	<strong><?php _e( 'State', 'swedbank-pay-woocommerce-payments' ); ?>
+		: </strong> <?php echo esc_html( $info['payment']['state'] ); ?>
+	<br/>
+	<?php if ( $gateway->core->canCapture( $order->get_id() ) ) : ?>
 		<button id="swedbank_pay_capture"
 				data-nonce="<?php echo wp_create_nonce( 'swedbank_pay' ); ?>"
 				data-payment-id="<?php echo esc_html( $payment_id ); ?>"
 				data-order-id="<?php echo esc_html( $order->get_id() ); ?>">
-			<?php _e( 'Capture Payment', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>
+			<?php _e( 'Capture Payment', 'swedbank-pay-woocommerce-payments' ); ?>
 		</button>
 	<?php endif; ?>
 
-	<?php if ( isset($info['payment']['remainingCancellationAmount']) && (float) $info['payment']['remainingCancellationAmount'] > 0.1 ): ?>
+	<?php if ( $gateway->core->canCancel( $order->get_id() ) ) : ?>
 		<button id="swedbank_pay_cancel"
 				data-nonce="<?php echo wp_create_nonce( 'swedbank_pay' ); ?>"
 				data-payment-id="<?php echo esc_html( $payment_id ); ?>"
 				data-order-id="<?php echo esc_html( $order->get_id() ); ?>">
-			<?php _e( 'Cancel Payment', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>
+			<?php _e( 'Cancel Payment', 'swedbank-pay-woocommerce-payments' ); ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (like esc_html_e() or esc_attr_e()), found '_e'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)